src/v2/api/index.md

```diff
@@ -2231,115 +2231,115 @@ type: api
+  `ref` es utilizado para registrar una referencia a un elemento a un componente hijo. La referencia será registrada bajo el objeto `$refs` del componente padre. Si se utiliza en un elemento "plano" del DOM, la referncia será a dicho elemento; si se utiliza en un componente hijo, la referencia será a la instancia de dicha componente:
 
   ``` html
-  <!-- vm.$refs.p will be the DOM node -->
-  <p ref="p">hello</p>
+  <!-- vm.$refs.p será el nodo del DOM -->
+  <p ref="p">hola</p>
 
-  <!-- vm.$refs.child will be the child component instance -->
+  <!-- vm.$refs.child será la intancia del componente hijo -->
   <child-component ref="child"></child-component>
   ```
 
-  When used on elements/components with `v-for`, the registered reference will be an Array containing DOM nodes or component instances.
+  Cuando se utiliza con elementos/components con `v-for`, la referencia registrada será un arreglo que contendrá nodos del DOM o instancias de componentes.
 
-  An important note about the ref registration timing: because the refs themselves are created as a result of the render function, you cannot access them on the initial render - they don't exist yet! `$refs` is also non-reactive, therefore you should not attempt to use it in templates for data-binding.
+  Una nota importante sobre el tiempo de registro de referencias: dado que las mismas son creadas como resultado de la función de renderizado, usted no puede acceder a las mismas en la renderización inicial - estas no existen todavía! También, `refs` no es reactivo, por lo tanto, no intente utilizarlo en plantillas para enlace de datos (_data-binding_).
 
-- **See also:** [Child Component Refs](../guide/components.html#Child-Component-Refs)
+- **Vea también:** [Referencias a Componentes Hijos](../guide/components.html#Child-Component-Refs)
 
 ### slot
 
-- **Expects:** `string`
+- **Espera:** `string`
 
-  Used on content inserted into child components to indicate which named slot the content belongs to.
+  Utilizado en el contenido insertado dentro de componentes hijo para indicar a que _slot_ con nombre pertenece el contenido.
 
-  For detailed usage, see the guide section linked below.
+  Para su uso detallado, vea la guía referenciada abajo.
 
-- **See also:** [Named Slots](../guide/components.html#Named-Slots)
+- **Vea también:** [Slots con Nombre](../guide/components.html#Named-Slots)
 
 ### slot-scope
 
-> New in 2.5.0+
+> Nuevo en 2.5.0+
 
-- **Expects:** `function argument expression`
+- **Espera:** `function argument expression`
 
 - **Uso:**
 
   Used to denote an element or component as a scoped slot. The attribute's value should be a valid JavaScript expression that can appear in the argument position of a function signature. This means in supported environments you can also use ES2015 destructuring in the expression. Serves as a replacement for [`scope`](#scope-replaced) in 2.5.0+.
```

Comment:
You missed the translation of this paragraph :)

Reply:
Damn 🤦‍♂